### PR TITLE
docs(ripple): improve ripple documentation

### PR DESCRIPTION
**Pull request by SisIvanova**
Linked issues: IgniteUI/igniteui-angular-samples/issues/1875

### CODE REVIEW

File: en/components/ripple.md [hunk 1]
```diff
@@ -47,162 +40,159 @@ export class AppModule {}
...
+<div class="sample-container loading" style="height: 100px">
     <iframe seamless="" width="100%" height="100%" frameborder="0" data-src="{environment:demosBaseUrl}/interactions/ripple-sample-6" class="lazyload">
 </iframe></div>
 
 #### Centered Ripple Effect 
-The ripple effect starts from the position of the click event. You can change this behavior using [`igxRippleCentered`]({environment:angularApiUrl}/classes/igxrippledirective.html#centered) and setting the center of the element as origin.
+By default, the ripple effect starts from the position of the click event. You can change this behavior using the [`igxRippleCentered`]({environment:angularApiUrl}/classes/igxrippledirective.html#centered) property and setting the center of the element as origin.
 
 ```html
 <button igxButton="raised" igxRipple="white" igxRippleCentered="true">Centered</button>
```
Comment (simeonoff) [inline en/components/ripple.md]:
```suggestion
<button igxButton="raised" igxRipple="white" [igxRippleCentered]="true">Centered</button>
```

File: en/components/ripple.md [hunk 2]
```diff
@@ -47,162 +40,159 @@ export class AppModule {}
...
+We can use the [`igxRippleDuration`]({environment:angularApiUrl}/classes/igxrippledirective.html#rippleduration) property to change the duration of the ripple animation, which, by default, is set to 600 milliseconds.
 
 ```html
-<div class="parent-div" igxRipple="blue" igxRippleTarget=".child-div" igxRippleCentered="true">
-  Parent Div
-  <div class="child-div">
-    Child Div
-  </div>
-</div>
+<button igxButton="raised" igxRipple [igxRippleDuration]=2000>Click Me</button>
```
Comment (simeonoff) [inline en/components/ripple.md]:
```suggestion
<button igxButton="raised" igxRipple igxRippleDuration="2000">Click Me</button>
```

File: en/components/ripple.md [hunk 3]
```diff
@@ -47,162 +40,159 @@ export class AppModule {}
...
-$yellow-color: #ffcd0f;
+@include igx-css-vars($custom-ripple-theme);
+```
 
-$custom-palette: igx-palette(
-    $primary: $black-color,
-    $secondary: $yellow-color
-);
-```   
+#### Using mixins
```
Comment (simeonoff) [inline en/components/ripple.md]: This is technically wrong as `igx-css-vars` is also a mixin.
```suggestion
#### Using Component Theme Overrides
```

File: en/components/ripple.md [hunk 4]
```diff
@@ -47,162 +40,159 @@ export class AppModule {}
...
+$custom-ripple-theme: igx-ripple-theme(
+    $palette: $custom-ripple-palette,
+    $schema: $custom-ripple-schema
+);
+```
+
+Don't forget to include the themes in the same way as it was demonstrated above. 
 
 >[!NOTE]
   > A color that is set using the `igxRiple` directive, would take precedence from the one, set within a custom theme.
```
Comment (simeonoff) [inline en/components/ripple.md]: This is not needed. Palettes are explained in their own section. Demonstrating how schemas work without context adds more confusion.

File: en/components/toast.md [hunk 5]
```diff
@@ -220,7 +220,7 @@ $custom-light-schema: extend($light-schema,(
     igx-toast: $light-toast-schema
 ));
 
-// Defining toast with the global light schema
+// Defining toast with the custom light schema
 $custom-toast-theme: igx-toast-theme(
   $palette: $light-toast-palette,
   $schema: $custom-light-schema
```
Comment (simeonoff) [inline en/components/toast.md]: I don't know why we change the toast.md file in this PR. Please remove.

Comment (SisIvanova) [reply en/components/toast.md]: just spotted a minor mistake and wanted to fix it

**Pull request merged**